src/components/layout.js

```diff
@@ -55,6 +55,17 @@ class Template extends React.Component {
             property="og:description"
             content="/spacebot - A Slack Bot that helps you learn and explore space. Learn about astronomy, NASA, mars rovers, and more."
           />
+          <meta property="og:image" content="https://raw.githubusercontent.com/mikelax/spacebot-site/master/src/assets/images/website-icon.png" />
```

Comment:
@sonimadhuri  thanks for working on this, we will have to make a few additional change to get the correct URL to use for this tag and the corresponding `twitter:image` url. 

We don't want to link to the github URL, but rather the generated URL should be an absolute URL based on the slashspacebot.com domain. 

* Add a `siteUrl` attribute to the `siteMetadata` attribute in the gatsby-config.js file. The value should be:   `https://slashspacebot.com`
* git mv the file `website-icon.png to the static folder. Update the value for the `icon` attribute in gatsby-config.js to be: `static/website-icon.png`
* In layout.js, you can use [graphql static query, tutorial here](https://www.gatsbyjs.org/tutorial/part-four/#use-a-staticquery) to get the `siteUrl` value with `/website-icon.png`. The resulting value for these should be:  `https://slashspacebot.com/website-icon.png`

Reply:
Hey @mikelax I'm on this, But I am pretty new to graphql

Reply:
Ok cool. Let me know if you have any questions or if you want me to push a little sample for the graphql query, etc.